### PR TITLE
Adapt Import / Export For Events

### PR DESCRIPTION
**Pull request by jdemangeon**
v1 example

Peek 14-05-2020 11-55

### CODE REVIEW

File: src/screens/simulation/Simulation.js [hunk 1]
```diff
@@ -220,6 +216,20 @@ const Simulation = () => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [JSON.stringify(parameters)]);
 
+    const handleImport = ({ version, data }) => {
+        if (version === 1) {
+            // There's no events, everything is in "data" on v1
+            setParameters(data);
```
Comment (Kmaschta) [inline src/screens/simulation/Simulation.js:222]: I'm not sure about this ...
In V1, data was a list of timeframes (it was broken with the introduction of the event tab)

File: src/screens/simulation/configuration/ConfigurationDrawer.js [hunk 2]
```diff
@@ -82,7 +82,7 @@ const ConfigurationForm = ({
                                     }}
                                 >
                                     <Tab label="configuration" />
-                                    <Tab label="évènements" />
+                                    <Tab label={`évènements (${Object.keys(events).length})`} />
```
Comment (Kmaschta) [inline src/screens/simulation/configuration/ConfigurationDrawer.js:85]: :+1:

File: src/screens/simulation/Simulation.js (hunk 1 above)
Comment (Kmaschta) [reply src/screens/simulation/Simulation.js:222]: I would prefer to display an error message than import wrong data

Comment (jdemangeon) [reply src/screens/simulation/Simulation.js:222]: I've tested and it's works, what do you mean exactly?

**Pull request merged**